testify/test_case.py

```diff
@@ -66,12 +68,8 @@ def _cmp_str(instance):
         """Return a canonical representation of a TestCase for sorting and hashing."""
         return "%s.%s" % (instance.__module__, instance.__name__)
 
-    def __cmp__(cls, other):
```

Comment:
nah, a remnant of past times I believe.  It was getting used in py2 for ==, but reference equality for classes is usually good enough :) in py3 it was causing flake8 to balk at `cmp`